server/Opendatacam.js

```diff
@@ -701,8 +697,10 @@ module.exports = {
     return Opendatacam.trackerDataForLastFrame;
   },
 
-  startStreamingData(sse) {
-    Opendatacam.sseConnexion = sse;
+  startStreamingData(res) {
```

Comment:
Should we rename this to something like "addStreamClient" to make it clear that this method can be called at any time (also on an already running stream)?